src/components/ha-selector/ha-selector.ts

```diff
@@ -59,6 +62,12 @@ export class HaSelector extends LitElement {
     return Object.keys(this.selector)[0];
   }
 
+  protected willUpdate(changedProps: PropertyValues) {
+    if (changedProps.has("selector") && this.selector) {
+      LOAD_ELEMENTS[this._type]();
```

Comment:
```suggestion
      LOAD_ELEMENTS[this._type]?.();
```
In case of no existing type 🙂 (I created some custom selector in mushroom 😅)